include/bluetooth/audio/audio.h

```diff
@@ -239,6 +250,10 @@ enum bt_audio_location {
 	BT_AUDIO_LOCATION_FRONT_RIGHT_WIDE = BIT(25),
 	BT_AUDIO_LOCATION_LEFT_SURROUND = BIT(26),
 	BT_AUDIO_LOCATION_RIGHT_SURROUND = BIT(27),
+	BT_AUDIO_LOCATION_RFU1 = BIT(28),
+	BT_AUDIO_LOCATION_RFU2 = BIT(29),
+	BT_AUDIO_LOCATION_RFU3 = BIT(30),
+	BT_AUDIO_LOCATION_RFU4 = BIT(31),
```

Comment:
Does it make sense to define RFU values? We haven't done that in any other public BT API

Reply:
They are defined as RFU, hence indicates people should not use them for their own defined location as they would then risk issues with future updates to the specification.
But I'll remove if we have not done this for the other areas of LE audio.

Reply:
> But I'll remove if we have not done this for the other areas of LE audio.

Or BT in general :)

We can define these in an internal header file and verify that they are not set.